entries/c/calpoly.edu.json

```
@@ -0,0 +1,26 @@
+{
+    "Cal Poly": {
+        "domain": "calpoly.edu",
+        "img": "calpoly.edu.png",
+        "tfa": [
+            "sms",
+            "call",
+            "u2f",
+            "custom-hardware",
+            "custom-software"
+        ],
+        "custom-hardware": [
+            "Duo token"
```

Most other uni sites seem to use this name for the token.
```suggestion
            "Duo D100 Token"
```